src/behavior.ts

```diff
@@ -67,6 +65,23 @@ export interface BehaviorOptions extends IBehaviorOptions<SunburstDataPoint> {
 export class Behavior implements IInteractiveBehavior {
     private options: BehaviorOptions;
 
+    private select(d:HierarchyRectangularNode<SunburstDataPoint>, selectionHandler: ISelectionHandler, onSelect: (dataPoint: SunburstDataPoint) => void, event: MouseEvent | KeyboardEvent) {
+        selectionHandler.handleSelection(d.data, event.ctrlKey);
+        event.stopPropagation();
+
+        if (onSelect) {
+            onSelect(d.data);
+        }
+    }
+
+    private clear(selectionHandler: ISelectionHandler, onSelect: (dataPoint: SunburstDataPoint) => void) {
+        selectionHandler.handleClearSelection();
+
+        if (onSelect) {
+            onSelect(null);
```

Comment:
please explain this onSelect call

Reply:
It does the same thing as it did in the previous code before being refactored into the clear() function:

It let's the visual know to render without anything in the selected state.